src/store/components/ItemPage/index.tsx

```diff
@@ -150,4 +158,15 @@ const ItemPage: React.FC<ItemPageProps> = (props) => {
   );
 };
 
-export default connect(null, { addToCart })(ItemPage);
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  redirect: (rejectRoute: string) => {
+    dispatch(replace(rejectRoute));
+  },
+  addToCart,
+});
+
+const mapStateToProps = (state: { [key: string]: any }) => ({
+  isAdmin: state.auth.admin,
```

Comment:
I double checked the portal account role specifications and it looks like store administrator roles also need full admin access only on the store UI so we have to change the logic to include `store_manager` too. 

Let's describe the logic better by renaming the variable to something like `isStoreAdmin` or `canViewHiddenItems` to demonstrate what the boolean is for more accurately

Full Description of Roles here if you want to understand the different scenarios better: https://github.com/acmucsd/membership-portal/blob/master/README.md#testing-information